README.md

```diff
@@ -131,17 +131,17 @@ Create a `secretproviderclasses` resource to provide provider-specific parameter
 
 1. Update your [linux deployment yaml](examples/nginx-pod-secrets-store-inline-volume-secretproviderclass.yaml) or [windows deployment yaml](examples/windows-pod-secrets-store-inline-volume-secretproviderclass.yaml) to use the Secrets Store CSI driver and reference the `secretProviderClass` resource created in the previous step. 
 
-        If you did not change the name of the secretProviderClass previously, no changes are needed.
+      If you did not change the name of the secretProviderClass previously, no changes are needed.
```

Comment:
While you are here, can you also add a note under `Prerequisites` at the top of this readme to point users to key vault flex vol if they are on 1.15 and below.

Reply:
@ritazh 1.15 is no longer supported. Adding a note to point to flexvol might be confusing because the solution is deprecated.